blocks/authAPI/views.py

```diff
@@ -9,7 +9,9 @@
 from django.http import HttpResponseNotFound
 from djoser import utils
 from djoser.serializers import TokenSerializer
```

Comment:
```suggestion
from djoser.serializers import TokenSerializer, PasswordResetConfirmSerializer
```
add PasswordResetConfirmSerializer here